apps/bookmark/js/bookmark_editor.js

```diff
@@ -37,6 +38,9 @@ var BookmarkEditor = {
 
     this.bookmarkTitle.value = this.data.name || '';
 
+    this.bookmarkURL.href = this.data.url;
+    this.bookmarkURL.innerHTML = this.data.url;
```

Comment:
`this.data` may come from any arbitrary app that wishes to create a bookmark. Using untrusted input with `innerHTML` seems like a bad idea.

Let's use `textContent` instead!